Assets/Scripts/Unity/UserController.cs

```diff
@@ -0,0 +1,30 @@
+﻿// <copyright file="UserController.cs" company="Delft University of Technology">
+// Copyright (c) Delft University of Technology. All rights reserved.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Meta;
+using UnityEngine;
+
+[SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1400:AccessModifierMustBeDeclared", Justification = "Reviewed.")]
+
+/// <summary>
+///  UserController keeps track of the user and its behavior.
+/// </summary>
+public class UserController : MonoBehaviour
```

Comment:
Yes, but it will still be used and is used in other branches, so I didn't remove it.